### DISCUSSION TITLE
Sharing a context with a third party library generates a lot of warning messages in Chrome

### DISCUSSION THREAD

**Discussion by koji117** [pixijs/pixijs#7467] [General]
Hi, I came from the discussion in the following forum.
https://www.html5gamedevs.com/topic/47221-use-a-webgl-context-from-a-third-party-library/

### Expected Behavior
Pixijs and a third party library can share the same webgl context without generating any warning messages in Chrome.

### Current Behavior
Sharing a context with a third party library generates a lot of warning messages such as below in Chrome (Chrome only)
RENDER WARNING: there is no texture bound to the unit x

### Possible Solution
This seems to be happening cause pixijs tries to render a texture that is no yet fully loaded. 
Three.js has a way to avoid this by calling onLoad() method. 

### Steps to Reproduce
Please have a look at the demo below
https://codepen.io/SKoji/pen/QWdeBzJ

### Environment

- **`pixi.js` version**:  6.0.2
- **Browser & Version**:  Chrome 90.0.4430.93 
- **OS & Version**:  macOS Big Sur 11.3
- **Running Example**:  https://codepen.io/SKoji/pen/QWdeBzJ

**Comment by eXponenta** [comment]
Not reproduced for Win10.

**Comment by koji117** [comment]
Just tried with Windows 10 and yes you are right. The issue cannot be reproduced with Windows 10 which actually makes it more strange.. So these warning messages are only generated when you use (1) macOS and (2) Google Chrome.

**Comment by bigtimebuddy** [comment]
Going to convert this to a discussion as issues isn't the right place getting help to debug projects. Generally, third party library integrations isn't something we support, but I'm sure folks might have some input who've had to shared context with other WebGL libraries.
